modules/twinklefluff.js

```diff
@@ -50,17 +57,11 @@ Twinkle.fluff = {
 				var list = $("#mw-content-text").find("ul li:has(span.mw-uctop)");
 
 				var revNode = document.createElement('strong');
-				var revLink = document.createElement('a');
-				revLink.appendChild( spanTag( 'Black', '[' ) );
-				revLink.appendChild( spanTag( 'SteelBlue', 'rollback' ) );
-				revLink.appendChild( spanTag( 'Black', ']' ) );
+				var revLink = buildLink('SteelBlue', 'rollback' );
```

Comment:
Nitpick -- I don't really care about `( code )` vs. `(code)` but you probably didn't mean to put `(code )` (with the trailing space). One day we should add some linter rules and unify all this. I think there are utilities to automatically update the code, too.

Reply:
Nice catch!  I try and match the nearby style, but (mostly because of perl stuff) I [do care](#500), so thanks.